LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/WebSocketWriterRegistry.cs

```diff
@@ -113,6 +120,7 @@ public IWebSocketWriterHandle<TMessage> Register(TKey key, IWebSocketWriter<TMes
             {
                 if (this.sockets.TryGetValue(key, out var current))
                 {
+                    this.stationConnectivityLostCounter?.Add(1);
```

Comment:
shouldn't we track the key/station eui here?

Reply:
This is why we have the `RegistryMetricTagBag`. The station EUI is made available through ambient environment and gets picked up by the exporters when raising the metrics. With that, we don't have to explicitly attach the Gateway ID to every metric that we raise. It's essentially the same technique that is used by logging scopes.